app/models/application.rb

```diff
@@ -1,4 +1,9 @@
 class Application < ApplicationRecord
   belongs_to :user
   belongs_to :pet
+
+  validates :date, presence: true
+  validates :approved, inclusion: { in: [true, false] }
```

Comment:
I'm not sure you need this as "approved" is already boolean? I think the inclusion stuff is more to do with if you want a dropdown menu of a particular list of strings (so what we had for video games, board games etc)
Not sure what kind of validation you need for a boolean (maybe a default false but that probably goes in the migration) so maybe check with a TA on this